ui/themes/OBAClassicApplicationUI.m

```diff
@@ -55,11 +57,29 @@ - (instancetype)init {
 
         _tabBarController.viewControllers = @[_mapNavigationController, _recentsNavigationController, _bookmarksNavigationController, _infoNavigationController];
         _tabBarController.delegate = self;
+
+        [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(refreshMessageBadge:) name:ApptentiveMessageCenterUnreadCountChangedNotification object:nil];
     }
 
     return self;
 }
 
+- (void)dealloc {
+    [[NSNotificationCenter defaultCenter] removeObserver:self name:ApptentiveMessageCenterUnreadCountChangedNotification object:nil];
```

Comment:
wow! this is amazing! http://useyourloaf.com/blog/unregistering-nsnotificationcenter-observers-in-ios-9/